tests/ui/conftest.py

```diff
@@ -49,12 +59,58 @@ def jwt_secret(base_url, variables):
 
 
 @pytest.fixture
-def initial_data(transactional_db):
+def initial_data():
     from olympia.amo.tests import addon_factory, user_factory
     from olympia.addons.models import AddonUser
+    from olympia.landfill.collection import generate_collection
+    from olympia.constants.applications import APPS
 
     for x in range(10):
         AddonUser.objects.create(user=user_factory(), addon=addon_factory())
+        generate_collection(addon_factory(), APPS['firefox'])
+
+
+@pytest.fixture
+def gen_addons():
+    from olympia.amo.tests import addon_factory
+    from olympia.constants.applications import APPS
+    from olympia.landfill.collection import generate_collection
+
+    for x in range(80):
```

Comment:
If `x` is unused, call it `_`. See https://hackernoon.com/understanding-the-underscore-of-python-309d1a029edc